### ISSUE TITLE
IDKR native aspect ratio not working.

### ISSUE THREAD

**Issue by Rekenem** [idkr-client/idkr#53]
everytime i use idkr the client freezes and ive noticed that my aspect ratio is supposed to be 1366 X 768  but it comes up as like 1360 X 706 or some weird stuff like that

**Comment by Rekenem** [comment]
can anyone help?

**Comment by Mixaz017** [comment]
Please provide more information. What exactly is not working? What is "idkr native aspect ratio"? Are you using idkr on fullscreen or windowed?

**Comment by Rekenem** [comment]
![Screenshot (1)](https://user-images.githubusercontent.com/82395284/148665851-86fc02ea-b650-4b40-adc4-8caf0cfee987.png)
![Screenshot (2)](https://user-images.githubusercontent.com/82395284/148665855-f82fb7a6-cc27-4451-9d9a-d94a10caa3ce.png)
i loaded up idkr twice and my native aspect ratio keeps changing for some reason i play in fullscreen mode

**Comment by Rekenem** [comment]
by native aspect ratio i mean the aspect ratio that IDKR sets when i load in its not a preset

**Comment by Mixaz017** [comment]
I'm sure that the placeholder displayed in Aspect Ratio setting does NOT update in real time. Which means it does not always represent your window size, it only displays your window size when the setting was initialized (which is usually when the game loads). I'm not sure about why your initial resolution is inconsistent, but it might be because your display resolution is smaller than the window size idkr initialize windows at (1600x900 including UI, meaning the actual content will be a bit smaller).

tldr: Don't use numbers displayed in the Aspect Ratio setting as your current resoluton.
